src/Stoa.ts

```diff
@@ -293,9 +297,14 @@ class Stoa extends WebService {
         this.app.get("/search/hash/:hash", isBlackList, this.searchHash.bind(this));
         this.app.get("/proposals/", isBlackList, this.getProposals.bind(this));
         this.app.get("/proposal/:proposal_id", isBlackList, this.getProposalById.bind(this));
+        this.app.get("/proposal/voting-details/:proposal_id", isBlackList, this.getVotingDetails.bind(this));
         this.app.get("/validator/reward/:address", isBlackList, this.getValidatorReward.bind(this));
-        this.app.get("/convert-to-usd", isBlackList, this.convertToUSD.bind(this));
+        this.app.get("/validator/ballot/:address", isBlackList, this.getValidatorBallots.bind(this));
+        this.app.get("/convert-to-currency", isBlackList, this.convertToCurrency.bind(this));
         this.app.get("/txhash/:utxo", isBlackList, this.getTransactionHash.bind(this));
+        this.app.get("/validator/missed-blocks/:address", isBlackList, this.getValidatorMissedBlocks.bind(this));
+        this.app.get("/block/validators", isBlackList, this.getBlockValidators.bind(this));
```

Comment:
If you need a paging function, please implement a new additional function.
`getBlockValidators` Do not modify this function because there is a service we are using.

Reply:
getBlockValidators is not modified

Reply:
This method will return all block validators if pagination is not provided